codegen/templates/vec.rs.tera

```diff
@@ -825,6 +825,49 @@ impl {{ self_t }} {
             }
         {% endif %}
     }
+
+    /// Returns a bitmask with the lowest {{ dim }} bits set to the sign bits from the elements of `self`.
+    ///
+    /// A negative element results in a `1` bit and a positive element in a `0` bit.  Element `x` goes
+    /// into the first lowest bit, element `y` into the second, etc.
+    #[inline]
+    pub fn sign_bits(self) -> u32 {
```

Comment:
I think perhaps `is_negative_bitmask` would be a better name, `sign` can be positive or negative and I already use `mask` in a few places to refer to a vector mask (e.g. `BVecN`).